package.json

```diff
@@ -243,6 +243,7 @@
     "@types/bluebird": "^3.1.1",
     "@types/boom": "^7.2.0",
     "@types/chance": "^1.0.0",
+    "@types/cheerio": "^0.22.10",
```

Comment:
This addition looks fine, although I don't see explicit usage of `cheerio` in our code, just pointing it out in case you don't want it added. Kind of curious what required the types.

Reply:
I was getting a type error when I used the enzyme-to-json library to create a snapshot. This was the easy solution to avoid a type error without updating all the libraries